From: Michael Olbrich <m.olbrich@pengutronix.de>
Date: Thu, 20 Mar 2014 10:12:07 +0100
Subject: [PATCH] make it work with automake 1.14

pkglibdir has special meaning and newer automake version don't allow
installer DATA here.

Signed-off-by: Michael Olbrich <m.olbrich@pengutronix.de>
---
 stage1/Makefile.am | 6 +++---
 stage2/Makefile.am | 8 ++++----
 2 files changed, 7 insertions(+), 7 deletions(-)

diff --git a/stage1/Makefile.am b/stage1/Makefile.am
index 0afc285..d850e10 100644
--- a/stage1/Makefile.am
+++ b/stage1/Makefile.am
@@ -1,7 +1,7 @@
-pkglibdir = $(libdir)/$(PACKAGE)/$(host_cpu)-$(host_vendor)
-nodist_pkglib_DATA = stage1
+libpkgdir = $(libdir)/$(PACKAGE)/$(host_cpu)-$(host_vendor)
+nodist_libpkg_DATA = stage1
 
-CLEANFILES = $(nodist_pkglib_DATA)
+CLEANFILES = $(nodist_libpkg_DATA)
 
 # We can't use builtins or standard includes.
 AM_CCASFLAGS = $(STAGE1_CFLAGS) -fno-builtin -nostdinc
diff --git a/stage2/Makefile.am b/stage2/Makefile.am
index 24b07e2..aee1563 100644
--- a/stage2/Makefile.am
+++ b/stage2/Makefile.am
@@ -27,12 +27,12 @@ libgrub_a_CFLAGS = $(GRUB_CFLAGS) -I$(top_srcdir)/lib \
 	-DUSE_MD5_PASSWORDS=1 -DSUPPORT_SERIAL=1 -DSUPPORT_HERCULES=1
 
 # Stage 2 and Stage 1.5's.
-pkglibdir = $(libdir)/$(PACKAGE)/$(host_cpu)-$(host_vendor)
+libpkgdir = $(libdir)/$(PACKAGE)/$(host_cpu)-$(host_vendor)
 
 EXTRA_PROGRAMS = nbloader.exec pxeloader.exec diskless.exec
 
 if DISKLESS_SUPPORT
-pkglib_DATA = stage2 stage2_eltorito e2fs_stage1_5 fat_stage1_5 \
+libpkg_DATA = stage2 stage2_eltorito e2fs_stage1_5 fat_stage1_5 \
 	ffs_stage1_5 iso9660_stage1_5 jfs_stage1_5 minix_stage1_5 \
 	reiserfs_stage1_5 ufs2_stage1_5 vstafs_stage1_5 xfs_stage1_5 \
 	nbgrub pxegrub
@@ -43,7 +43,7 @@ noinst_PROGRAMS = pre_stage2.exec start.exec start_eltorito.exec \
 	reiserfs_stage1_5.exec ufs2_stage1_5.exec vstafs_stage1_5.exec \
 	xfs_stage1_5.exec nbloader.exec pxeloader.exec diskless.exec
 else
-pkglib_DATA = stage2 stage2_eltorito e2fs_stage1_5 fat_stage1_5 \
+libpkg_DATA = stage2 stage2_eltorito e2fs_stage1_5 fat_stage1_5 \
 	ffs_stage1_5 iso9660_stage1_5 jfs_stage1_5 minix_stage1_5 \
 	reiserfs_stage1_5 ufs2_stage1_5 vstafs_stage1_5 xfs_stage1_5
 noinst_DATA = pre_stage2 start start_eltorito
@@ -105,7 +105,7 @@ else
 BUILT_SOURCES = stage2_size.h
 endif
 
-CLEANFILES = $(pkglib_DATA) $(noinst_DATA) $(BUILT_SOURCES)
+CLEANFILES = $(libpkg_DATA) $(noinst_DATA) $(BUILT_SOURCES)
 
 stage2_size.h: pre_stage2
 	-rm -f stage2_size.h
